internal/scrub/scrub.go

```diff
@@ -0,0 +1,174 @@
+// This package scrubs objects of customer information to pass to logging
```

Comment:
I'd almost say maybe this makes sense in /internal/log instead to keep them close together (in a scrub.go file)

Reply:
I was debating adding a flag in the context, but that, along with setting a flag on the logger, wouldnt work because a lot of places create a new logger from scratch rather than inherit it from parent functions.
I do like the idea of moving it to `internal/log` tho